src/components/Summary/PersonalDataFormStyles.ts

```diff
@@ -4,6 +4,10 @@ import { Container, Item } from "../../common/styles/componentsStyles";
 export const FormItem = styled(Item)`
   flex-direction: column;
   margin: 0 1rem;
+
+  @media (max-width: 540px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme